packages/babel-plugin-transform-for-of/src/index.js

```diff
@@ -35,11 +33,14 @@ export default declare((api, options) => {
     options.assumeArray ??
     // Loose mode is not compatible with 'assumeArray', so we shouldn't read
     // 'iterableIsArray' if 'loose' is true.
-    (!loose && api.assumption("iterableIsArray"));
+    (!options.loose && api.assumption("iterableIsArray"));
```

Comment:
nit: could use `loose` instead of `options.loose` here.

Reply:
Actually, I am explicitly trying to use `options.loose` in my PRs so that ESLint tells me that `loose` is unused and I'm sure I didn't miss proposing any assumption 😬